src/coreclr/jit/fgbasic.cpp

```diff
@@ -486,9 +486,9 @@ void Compiler::fgReplaceSwitchJumpTarget(BasicBlock* blockSwitch, BasicBlock* ne
             //
             // Create the new edge [newTarget from blockSwitch]
             //
-            flowList* const newEdge = fgAddRefPred(newTarget, blockSwitch);
+            FlowEdge* const newEdge = fgAddRefPred(newTarget, blockSwitch);
 
-            // Now set the correct value of newEdge->flDupCount
+            // Now set the correct value of newEdge's lDupCount
```

Comment:
nit: DupCount? or m_dupCount?

Reply:
Oops. Will fix this as part of an upcoming PR.